internal/controllers/clusterextension_controller_test.go

```diff
@@ -303,6 +369,7 @@ func TestClusterExtensionResolutionAndUnpackSuccessfulApplierFails(t *testing.T)
 	t.Log("By checking the status fields")
 	expectedBundleMetadata := ocv1alpha1.BundleMetadata{Name: "prometheus.v1.0.0", Version: "1.0.0"}
 	require.Equal(t, expectedBundleMetadata, clusterExtension.Status.Resolution.Bundle)
+	require.Equal(t, ocv1alpha1.BundleMetadata{Name: "prometheus.v1.0.0", Version: "1.0.0"}, clusterExtension.Status.Resolution.Bundle)
```

Comment:
This line seems like a duplicate of the one above it, just instead it is manually building the bundle metadata instead of using the variable on line 370

Reply:
will remove it